specs/gsb-api.yaml

```
@@ -0,0 +1,134 @@
+openapi: 3.0.3
+info:
+  version: 0.1.0
+  title: Golem GSB API
+  description: "Yagna GSB API"
+servers:
+  - url: /gsb-api/v1
+
+security:
+  - app_key: []
+
+tags:
+  - name: requestor
+    description: Requestor part
+
+paths:
+  /services:
+    post:
+      tags:
+        - requestor
+      operationId: bindServices
+      summary: Binds new GSB services.
+      responses:
```

Presumably following request body specification is required in `bindServices` call:
```suggestion
      requestBody:
        $ref: '#/components/requestBodies/ServicesRequest'
      responses:
```